src/components/features/forms/CheckboxRoles.tsx

```diff
@@ -1,190 +1,97 @@
-                  name="createZones"
-                  checked={checkedItems.createZones}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Създаване на зони')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="readZones"
-                  checked={checkedItems.readZones}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Четене на зони')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="editZones"
-                  checked={checkedItems.editZones}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Редактиране на зони')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="deleteZones"
-                  checked={checkedItems.deleteZones}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Триене на зони')}
-            />
-          </FormControl>
-        </Box>
-
-        <Box flex="1">
-          <FormControl component="fieldset">
-            <Typography variant="h6">{translate('Доставки')}</Typography>
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="createDeliveries"
-                  checked={checkedItems.createDeliveries}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Създаване на доставки')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="readDeliveries"
-                  checked={checkedItems.readDeliveries}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Четене на доставки')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="editDeliveries"
-                  checked={checkedItems.editDeliveries}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Редактиране на доставки')}
-            />
-            <FormControlLabel
-              control={
-                <Checkbox
-                  name="deleteDeliveries"
-                  checked={checkedItems.deleteDeliveries}
-                  onChange={handleCheckboxChange}
-                />
-              }
-              label={translate('Триене на доставки')}
-            />
-          </FormControl>
+      <Box display="flex" justifyContent="space-between" flexDirection={'column'}>
+        <Box display={'flex'}>
+          {firstColumns.map((col) => (
```

Comment:
use grids to do what you want to do here, because now if we add another type of role it will go to second columns and break and when more columns are added we will need to create more variables and etc.
https://mui.com/material-ui/react-grid/